ynr/settings/base.py

```diff
@@ -333,7 +334,7 @@ def root(*path):
 Q_CLUSTER = {
     "name": "DjangORM",
     "workers": 4,
-    "timeout": 90,
+    "timeout": None,
```

Comment:
`None` disables timeouts completely - tasks can run forever. I don't have an intuition for what a sensible value for a timeout is here, given the tasks we are starting with here. Any suggestions?

Reply:
Do we get an exception raised to Sentry if we hit the timeout? I think 5 minutes should be the maximum value here.